main.tf

```diff
@@ -5,8 +5,24 @@ locals {
     elb_name         = null
     container_port   = var.port_gateway
   }] : []
-  container_cpu         = var.container_cpu != null ? var.container_cpu : data.aws_ssm_parameter.container_cpu[0].value
-  docker_labels         = merge({}, var.docker_labels)
+  container_cpu = var.container_cpu != null ? var.container_cpu : data.aws_ssm_parameter.container_cpu[0].value
+  docker_labels = merge(var.docker_labels, {
```

Comment:
i would rather append vars than prepend them. In doubt you might wanna rather overwrite a value in this map instead of having your passed var overwritten by values in this fixed map.